src/plugins/ffe/ffe.py

```diff
@@ -23,7 +22,7 @@
 
 
 from web.controllers.admin.base_event_admin_controller import BaseEventAdminWebContext
-import web.controllers.base_controller as WebContextModule
+import web.controllers.base_controller as web_context_module
```

Comment:
I don't think the module is necessary here, I think the classes used can be directly imported without losing any clarity: 
 
```suggestion
from web.controllers.base_controller import BaseController, WebContext
```

Reply:
Done.